### DISCUSSION TITLE
Bypass icloud activation

### DISCUSSION THREAD

**Discussion by RafaelNMeyer** [LukeZGD/Legacy-iOS-Kit#458] [Q&A] [answered]
Hey! I have an iPad 3 (9.3.5) and it is locked by iCloud :(. I want to know if there is any method I can get around!
I already restored it with the restore.sh script (I'm using arch linux), but it's still blocked :((

**Comment by userse31** [comment, marked as answer]
Boot up via SSH ramdisk, mount the partitions, and remove "setup.app" from the system partition.
